src/arm/Gtrace.c

```diff
@@ -251,6 +251,12 @@ trace_init_addr (unw_tdep_frame_t *f,
       && likely(dwarf_put (d, d->loc[UNW_ARM_R15], pc) >= 0)
       && likely((ret = unw_step (cursor)) >= 0))
     *f = c->frame_info;
+  #if defined (__QNX__)
+    if (unw_is_signal_frame (cursor))
+      {
```

Comment:
Wrong file. This patch should be in src/aarch64/Gtrace.c

Reply:
oops my bad. Gonna fix it

Reply:
✔️